packages/flutter/test/material/bottom_navigation_bar_theme_test.dart

```diff
@@ -257,6 +263,7 @@ void main() {
     expect(findFadeTransition, findsNothing);
     expect(_material(tester).elevation, equals(elevation));
     expect(_material(tester).color, equals(backgroundColor));
+    expect(tester.getTopLeft(find.text('Alarm')).dx, 535);
```

Comment:
Maybe a comment here why it should be at 535.

Reply:
That check wasn't really needed and it would have complicated to explain the derivation of 535, so I removed it.